Cargo.toml

```diff
@@ -7,11 +7,12 @@ members = [
     "src/evm-block-extractor",
     "src/evm-canister-client",
     "src/iceth-client",
-    "src/minter-did",
-    "src/minter-client",
     "src/iceth-client/test-canister",
+    "src/minter-client",
+    "src/minter-did",
     "src/register-evm-agent/src/register_evm_agent",
     "src/register-evm-agent/src/simple_canister",
+    "src/rpc-client",
```

Comment:
should be `ethereum_json_rpc_api_client`

Reply:
If you don't mind, I've changed it to `ethereum-json-rpc-client` to make it a bit shorter

Reply:
perfect